pkgs/development/libraries/libck/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ck";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "concurrencykit";
+    repo = "ck";
+    rev = version;
+    sha256 = "1w7g0y1n7jslca693fb8sanlfi1biq956dw6avdx6pf3c2s7l9jd";
+  };
+
+  dontDisableStatic = true;
+
+  meta = with stdenv.lib; {
+    description = "High-performance concurrency research library";
+    longDescription = ''
+      Concurrency primitives, safe memory reclamation mechanisms and non-blocking data structures for the research, design and implementation of high performance concurrent systems.
+    '';
+    license = licenses.asl20;
```

```suggestion
    license = with licenses; [ asl20 bsd2 ];
```

http://concurrencykit.org/ mentions license as "Simplified BSD License" which is another name for the 2-clause BSD license. https://github.com/concurrencykit/ck/blob/master/LICENSE has the 2-clause BSD license first but says that src/ck_hp.c also has the ASL2.0 license. :woman_shrugging:

Also the license and source code license headers seem to have a line "All Rights Reserved", I'm not really sure what that means but it seems ok to just ignore in my opinion given the bsd2 license text also always there.

---

good catch!